src/test/java/core/basesyntax/services/impl/ConverterMapToListImplTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.services.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ConverterMapToListImplTest {
+    private Fruit apple;
+    private Fruit kiwi;
+    private Map<Fruit, Integer> testStorage;
+
+    @Before
+    public void setUp() {
+        apple = new Fruit("apple");
+        kiwi = new Fruit("kiwi");
+        testStorage = Map.of(apple, 10, kiwi, 10);
+    }
+
+    @Test
+    public void convertMethodTest_Ok() {
+        List<String> expectedList = List.of("apple,10", "kiwi,10");
+        List<String> actualList = new ConverterMapToListImpl().convert(testStorage);
```

Comment:
looks like we can create `ConverterMapToListImpl` only once in method annotated BeforeClass